### PR TITLE
SISRP-8847 - CC Profile - Only show items that are currently production ready

### PR DESCRIPTION
**Pull request by christianv**
https://jira.berkeley.edu/browse/SISRP-8847

### CODE REVIEW

**Comment by ctweney** [comment]
Looks reasonable. Waiting to merge until @raydavis @johncrossman and @paulkerschen have a chance to review as well.

File: src/assets/templates/widgets/profile/emergency_contact.html [hunk 1]
```diff
@@ -4,18 +4,14 @@
   <div class="cc-page-widget-profile-section-content">
     <div data-ng-repeat="emergencyContact in emergencyContacts.content" class="cc-page-widget-profile-section-content-item">
       <div>
-        <strong data-ng-bind="emergencyContact.name.formattedName"></strong>
-        <span data-ng-bind-template="- {{emergencyContact.relationship.description}}"></span>
+        <strong data-ng-bind="emergencyContact.name"></strong>
         <span data-ng-if="emergencyContact.primary">(primary contact)</span>
```
Comment (johncrossman) [inline src/assets/templates/widgets/profile/emergency_contact.html]: Is the "(primary contact)" within 'span' intentional?

**Comment by johncrossman** [comment]
This LGTM. I posted one question.

File: src/assets/templates/widgets/profile/emergency_contact.html (hunk 1 above)
Comment (christianv) [inline src/assets/templates/widgets/profile/emergency_contact.html]: @johncrossman Yes, it's part of the UI design

**Comment by paulkerschen** [comment]
Good by me.

**Pull request merged**